website/src/main/java/com/google/step/servlets/DataServlet.java

```diff
@@ -64,7 +62,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         "name TEXT NOT NULL", 
         "link TEXT NOT NULL", 
         "about TEXT NOT NULL",
-        "class VARCHAR(255) NOT NULL");
+        "class VARCHAR(255) NOT NULL",
```

Comment:
Does VARCHAR(255) work for batches of 500 and more?

Reply:
So far so good